protocol/xmrtaker/swap_state_test.go

```diff
@@ -308,7 +313,10 @@ func TestSwapState_NotifyClaimed(t *testing.T) {
 	require.NoError(t, err)
 
 	// mine some blocks to get xmr first
-	_ = maker.GenerateBlocks(xmrmakerAddr.Address, 60)
+	err = maker.GenerateBlocks(xmrmakerAddr.Address, 60)
+	require.NoError(t, err)
+	err = maker.Refresh()
```

Comment:
After I switched the code to use dynamic `monero-wallet-rpc` instances, this test started failing with insufficient balance.  Before the change, the wallet would have had an existing balance created by other tests.  The change here that made the difference was the `Refresh`.  Mining 60 blocks works, but the wallet isn't syncing its balance often enough to know that its balance changed without the `Refresh`.  We probably need to audit the tests for every call to `GenerateBlocks` and see if a wallet refresh after it is warranted.  There are probably places in the tests that are mining more blocks than needed, because if you mine enough blocks the mining operation will overlap with a non-explicit wallet refresh.